core/examples/http_bench_json_ops/main.rs

```diff
@@ -67,25 +45,19 @@ impl TryFrom<std::net::TcpListener> for TcpListener {
   ) -> Result<Self, Self::Error> {
     tokio::net::TcpListener::try_from(std_listener).map(|tokio_listener| Self {
       inner: tokio_listener,
-      cancel: Default::default(),
     })
   }
 }
 
 struct TcpStream {
   rd: AsyncRefCell<tokio::net::tcp::OwnedReadHalf>,
   wr: AsyncRefCell<tokio::net::tcp::OwnedWriteHalf>,
-  // When a `TcpStream` resource is closed, all pending 'read' ops are
-  // canceled, while 'write' ops are allowed to complete. Therefore only
-  // 'read' futures are attached to this cancel handle.
-  cancel: CancelHandle,
```

Comment:
Why is this removed?

Reply:
Because cancellation is not used in the benchmark.